CONTRIBUTING.md

```diff
@@ -35,9 +35,8 @@ detailing the change that needs to be made, after the designs you made have been
 like if you wanted to work on an issue titled "re-design website buttons".
 1. First, you would assign the issue to yourself by commenting `/assign` on the issue
 2. Next, you would work on an initial draft of the design.
-3. Once you have a design you are happy with or that you want feedback on, you will leave a comment on the issue with the artefacts from the design (e.g. a PNG output of the 
-button design from Figma)
-4. The design leads for the working group will leave feedback on the design, and possible ask you to make some changes with comments on the issue.
-5. Once the design leads are happy with your design, they will let you know on the issue. 
+3. Once you have a design you are happy with or that you want feedback on, you will open a Pull Request to this repository with the artifacts included.
```

Comment:
We don't have any directories other than `designs` in this repo. I don't think we need to create them with this PR, but we should have the awareness to organize stuff. As I said, no issues for now.